_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,171 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can dispatch a variety of messages through various channels
+and varying formats alerting you to events in your environment.
+
+## Notification Types
+
+  * Inventory Summary: Count of the resources added in the latest inventory crawl.
+  * Violation Notifications: Individual violations that have been found from the latest scanner run. 
+  * Cloud SCC Findings: Violations converted to [Cloud SCC (Cloud Security Command Center)](https://cloud.google.com/security-command-center/) findings format, that's ingestable by Cloud SCC.
+  
+## Notification Channels
+
+  * Email
+  * Slack
+  * Cloud Storage
+
+## Notification Formats
+
+  * Human-readable data: CSV
+  * Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary
+
+This is a count of what resources have been crawled into inventory,
+and outputted to a Cloud Storage bucket.
+
+To configure how you want Notifier to send the Inventory Summary,
+follow the steps below
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+
+1. Navigate to the `notifier` > `inventory` > `summary` section.
+
+```yaml
+notifier:
+    inventory:
+      summary:
+        enabled: true
+        data_format: csv
+        gcs_path: gs://path_to_foo_bucket
+```
+
+* `enabled`: `true` or `false`. Enables/disables the notification.
```

Comment:
These configuration variables need documented as you did below....

+* `slack_webhook`
  * **Description**: This sends individual violations to a Slack channel via a Slack webhook.
  * **Valid values**: String
  * **Note**: See [this Slack documentation on how to generate a webhook](https://api.slack.com/incoming-webhooks).

Reply:
Done